src/main/java/frc/robot/subsystems/BuddyBalanceSubsystem.java

```diff
@@ -4,107 +4,187 @@
+    return Math.round(AbsoluteEncoder.getPositionRadians(encoder)*100)/100.0;
   }
 
   public boolean getIsDeployed() {
     return isDeployed;
   }
 
-  public void deployBuddyBalance() {
-    activateDeploy.set(kServoDeployedPos);
+  public void deployBuddyBalance() { // Removes servo locking mechanism, releasing gas shocks
+    deployServo1.set(kServoDeployedPos1);
+    deployServo2.set(kServoDeployedPos2);
     isDeployed = true;
+    buddyBalancePosEntry.setString("Deployed");
+    servoPositionEntry.setDouble(deployServo1.get());
+  }
+
+  public void returnServos() { // Returns servos to the docked position
+    deployServo1.set(kServoInitialPos);
+    deployServo2.set(kServoInitialPos);
+    buddyBalancePosEntry.setString("Docked");
+    servoPositionEntry.setDouble(deployServo1.get());
   }
 
   public void retrieveBuddy() { // Used to pick up the buddy robot while the lift is already underneath it
-    rightPIDController.setReference(kBalancedPosition, CANSparkMax.ControlType.kPosition);
-    leftPIDController.setReference(kBalancedPosition, CANSparkMax.ControlType.kPosition);
+    unifiedPIDController.setSetpoint(Units.degreesToRadians(kRetrievedAngle));
+    buddyBalancePosEntry.setString("Raised");
   }
 
   public void releaseBuddy() { // Used to set down the robot
-    rightPIDController.setReference(kDeployedPosition, CANSparkMax.ControlType.kPosition);
-    leftPIDController.setReference(kDeployedPosition, CANSparkMax.ControlType.kPosition);
+    unifiedPIDController.setSetpoint(Units.degreesToRadians(kDeployedAngle));
+    buddyBalancePosEntry.setString("Lowered");
+  }
+
+  public void updateMotors() {
+    rightMotor1.set(MathUtil.clamp(unifiedPIDController.calculate(getDesiredAngle()), -1, 1));
+    rightMotor2.set(MathUtil.clamp(unifiedPIDController.calculate(getDesiredAngle()), -1, 1));
+    leftMotor1.set(MathUtil.clamp(unifiedPIDController.calculate(getDesiredAngle()), -1, 1));
+    leftMotor2.set(MathUtil.clamp(unifiedPIDController.calculate(getDesiredAngle()), -1, 1));
+    rightMotorOutputEntry.setDouble(rightMotor1.get());
+    leftMotorOutputEntry.setDouble(leftMotor1.get());
+    setpointEntry.setDouble(unifiedPIDController.getSetpoint());
+  }
+
+  // These methods are used for JUnit testing only
+  public void close() throws Exception {
+    // This method will close all device handles used by this object and release any other dynamic memory.
+    // Mostly for JUnit tests
+    rightMotor1.close();
+    rightMotor2.close();
+    leftMotor1.close();
+    leftMotor2.close();
+    deployServo1.close();
+    deployServo2.close();
+    encoder.close();
+  }
+
+  public double getActualServoPosition1() {
+    return deployServo1.get();
+  }
+  public double getActualServoPosition2() {
+    return deployServo2.get();
+  }
+
+  public static double getDesiredServoPosition1() {
+    return kServoDeployedPos1;
+  }
+
+  public static double getDesiredServoPosition2() {
+    return kServoDeployedPos2;
+  }
```

Comment:
Since they are constants, you can just make kServoDeployedPos1 & 2 public instead of creating getter methods